kinesumer.go

```diff
@@ -491,22 +498,30 @@ func (k *Kinesumer) consumePipe(stream string, shard *Shard) {
 				continue
 			}
 
-			// Check point the sequence number.
-			ctx := context.Background()
-			ctx, cancel := context.WithTimeout(ctx, checkPointTimeout)
-			if err := k.stateStore.UpdateCheckPoint(ctx, stream, shard.ID, lastSequence); err != nil {
+			if err := k.commitCheckPoint(stream, shard, lastSequence); err != nil {
 				log.Err(err).
 					Str("stream", stream).
 					Str("shard id", shard.ID).
 					Str("missed sequence number", lastSequence).
 					Msg("kinesumer: failed to UpdateCheckPoint")
```

Comment:
We're doing same thing (error logging) if commit fails on every commit places, so we can move this logging code to `commitCheckPoint` method to remove redundant codes.